src/Sorts/SelectionSort.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title returns an array of sorted numbers
+ * @author Chiadikaobi https://github.com/Chiadikaobixo
```

Comment:
We should also use this syntax in another PR. Will be making an issue for it soon. 🙂
```suggestion
 * @author [Chiadikaobi](https://github.com/Chiadikaobixo)
```